packages/browserslist-config/test/index.test.js

```diff
@@ -0,0 +1,17 @@
+const browserslist = require( 'browserslist' );
+const config = require( '../' );
+
+beforeEach(() => {
+	jest.resetModules();
+});
+
+it( 'should export an array', () => {
+	expect( Array.isArray( config ) ).toBe( true );
+});
+
+it( 'should not contain invalid queries', () => {
+	jest.doMock( '@wordpress/browserslist-config', () => require( '../index' ), { virtual: true });
+
+	const result = browserslist(['extends @wordpress/browserslist-config']);
```

Comment:
Spacing should be fixed to match coding styles.

Reply:
I'm going to skip this change, for now, we need to get ESLint setup

The blocker to this is getting the changes from https://make.wordpress.org/core/2017/09/07/proposal-for-js-standards-revision-removing-array-function-whitespace-exceptions/ merged.

In me hacking in ESLint to run over this PR both the following return no ESLint errors:
• `const result = browserslist([ 'extends @wordpress/browserslist-config' ]);`
• `const result = browserslist(['extends @wordpress/browserslist-config']);`

And this does:
• `const result = browserslist( ['extends @wordpress/browserslist-config'] );`

I'm not sure why `space-in-parens` ESLint rule is throwing that, but I'll get it fixed over the weekend, or early next week, a public holiday here today so I'm going to head out 🏖